pyccl/neutrinos.py

```diff
@@ -12,8 +19,8 @@ def Omeganuh2(a, Neff, mnu, TCMB=2.725):
 
     Args:
         a (float): Scale factor, normalized to 1 today.
-        Neff (float): Number of relativistic neutrino species (NB: to all practical purposes, Neff is simply N_nu_mass)
-        mnu (float): Neutrino mass (in eV)
+        Neff (float): Number of massive neutrino species (NB: to all practical purposes, Neff is simply N_nu_mass)
```

Comment:
This comment confuses me -- I thought it was the effective number of relativistic neutrinos?

Reply:
This is a very good catch. We shouldn't be passing this in at all, it should be the lengths of mnu. I've fixed this. This is another API change, added to the changelog.